modules/twinkleblock.js

```diff
@@ -125,6 +126,10 @@ Twinkle.block.fetchUserInfo = function twinkleblockFetchUserInfo(fn) {
 			// Used later to check if block status changed while filling out the form
 			Twinkle.block.blockLogId = Twinkle.block.hasBlockLog ? data.query.logevents[0].logid : false;
 
+			Twinkle.block.userIsBot = userinfo.groupmemberships.map(function(e) {
```

Comment:
`userinfo.groupmemberships` is undefined when you blocking IP.

Reply:
Word!  So, that'd be, what, `!!userinfo.groupmemberships && userinfo.groupmemberships.map...`?  Mobile, will push tomorrow.

Reply:
Thanks for the note @Xi-Plus; restructured it in c3916f4